boards/xtensa/intel_s1000_crb/intel_s1000_crb_defconfig

```diff
@@ -18,6 +18,8 @@ CONFIG_3RD_LEVEL_INTERRUPTS=y
 CONFIG_CAVS_ICTL=y
 CONFIG_DW_ICTL=y
 
+CONFIG_BOOTLOADER_SRAM_SIZE=192
```

Comment:
remind me, where is this 192 coming from?

Reply:
This refers to the size we have reserved for the mcubootloader. 192kB.